extensions/index.js

```diff
@@ -5,7 +5,7 @@ const extensions = {};
 
 fs.readdirSync(__dirname).forEach(moduleDir => {
     const extStat = fs.statSync(path.join(__dirname, moduleDir));
-    if (extStat.isDirectory()) {
+    if (extStat.isDirectory() && moduleDir !== 'utils') {
```

Comment:
If we move everything from this `utils` directory to the global backbeat `lib/utils` dir, we can get rid of this special case.

Reply:
I would like to table this change for a refactor in the future, since we'll have to move around a lot of files